frontend/.babelrc

```diff
@@ -0,0 +1,13 @@
+{
+  "presets": [
+    [
+      "@babel/preset-env",
```

Comment:
This is also due to get obsolete. Rather avoid if possible.

Reply:
Which part? Couldn't find anything online.